File: src/main/java/com/sem/keeper/formatter/FormatterConfig.java
package com.sem.keeper.formatter;

import org.springframework.context.annotation.Configuration;
import org.springframework.format.FormatterRegistry;
import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;

@Configuration
public class FormatterConfig implements WebMvcConfigurer {

    public void addFormatters(FormatterRegistry registry) {
        registry.addFormatter(new LocalDateTimeFormatter());
        registry.addFormatter(new DurationFormatter());
    }
}

File: src/main/java/com/sem/keeper/formatter/FormatterConfig.kt
package com.sem.keeper.formatter

import org.springframework.context.annotation.Configuration
import org.springframework.format.FormatterRegistry
import org.springframework.web.servlet.config.annotation.WebMvcConfigurer

@Configuration
open class FormatterConfig : WebMvcConfigurer {
    override fun addFormatters(registry: FormatterRegistry) {
        registry.addFormatter(LocalDateTimeFormatter())
        registry.addFormatter(DurationFormatter())
    }
}